index.html

```diff
@@ -44,8 +44,8 @@ <h3 class="profession">I'am a Software Developer</h3>
               </ul>
           </div>
      </div>
-<section class="card1">
-    <div class="work-title grid1"><h1 id="work-title"> My Recent Works</h1></div>
+<div class="card1">
```

Comment:
- [ ] Good job so far 👍 For best semantic practices it would be better to use a `section` tag here instead of the generic `div` tag 👍